js-closures.md

```diff
@@ -10,9 +12,9 @@ function makeFunc() {
 }
 
 var myFunc = makeFunc();
-myFunc();
-```
-
-See [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures)
-
+myFunc();
+```
+
+See [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures)
+
 > tags: closure, javascript, js
```

Comment:
Did you notice this? Not sure if it was from your previous PR or if it was there before. Would that work?